radius-rails.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   #spec.add_dependency "font-awesome-rails"
-  spec.add_dependency "railties", ">= 3.2", "< 6.0"
+  spec.add_dependency "railties", ">= 3.2", "< 6.1"
```

Comment:
If we assume rails won't break railtie interface until a major version changes we could just tiddle-waka and avoid this dance for a while.

```suggestion
  spec.add_dependency "railties", ">= 3.2", "~> 6.0"
```

Reply:
Good call 👍